### DISCUSSION TITLE
Support isar store

### DISCUSSION THREAD

**Discussion by LasseRosenow** [zino-app/graphql-flutter#1286] [Ideas]
As of now the `graphql_flutter` client only supports hive as a store by default.

But even the [hive](https://github.com/hivedb/hive) mentions in its `README`: "Consider using [Isar](https://github.com/isar/isar), a Flutter database by the author of Hive that is superior in every way!"

So I was wondering, if `graphql_flutter` would like to implement more official stores like [Isar](https://github.com/isar/isar) for example.

**Comment by vincenzopalazzo** [comment]
we can take it in considederation
